docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -62,14 +64,14 @@ To expand the analyzer to a full screen, select the **Full Screen** icon above t
 [role="screenshot"]
 image::images/full-screen-analyzer.png[]
 
-The left panel contains a list of all processes related to the event, starting with the event chain's first process. **Analyzed Events**, as in the event you selected to analyze from either the events list or your timeline, are highlighted by a light blue outline around the cube.
+The left panel contains a list of all processes related to the event, starting with the event chain's first process. **Analyzed Events**, as in the event you selected to analyze from either the events list or your Timeline, are highlighted by a light blue outline around the cube.
 
 [role="screenshot"]
 image::images/process-list.png[]
 
 In the graphical view, you can:
 
-- Zoom in and out of the graphical view using the slider to the right of the timeline
+- Zoom in and out of the graphical view using the slider to the right of the Timeline
```

Comment:
My bad about this - timeline in this scenario is lowercase since we aren't talking about the feature. 

```suggestion
- Zoom in and out of the graphical view using the slider to the right of the timeline
```